test/example-types/filters/text.js

```diff
@@ -150,12 +149,12 @@ describe('text', () => {
           should: [
             {
               regexp: {
-                'description.untouched': '[Ll][Aa][Ss][Ee][Rr][Jj][Ee][Tt]',
+                'description.untouched': 'laserjet',
               },
             },
             {
               regexp: {
-                'description.untouched': '[Pp][Rr][Ii][Nn][Tt][Ee][Rr]',
+                'description.untouched': 'printer',
```

Comment:
These should have the case insensitive flag

Reply:
the tests pass and will fail if I add that. Not sure what you could mean here

Reply:
Based on this test, it looks like you're generating a query that isn't using the case insensitive flag but no longer manually does case-insensitivity with the letter brackets. If we leave this test as is, it shows that the behavior will change.